### PR TITLE
Prep for v1.23.0

### PR DESCRIPTION
**Pull request by odow**
A pretty quite month here really.

## Basic

 - [x] `version` field of `Project.toml` has been updated
       - If a breaking change, increment the MAJOR field and reset others to 0
       - If adding new features, increment the MINOR field and reset PATCH to 0
       - If adding bug fixes or documentation changes, increment the PATCH field

## Documentation

 - [x] Add a new entry to `docs/src/changelog.md`, following existing style

## Tests

 - [x] https://github.com/jump-dev/MathOptInterface.jl/actions/runs/7025896291

### CODE REVIEW

File: docs/src/changelog.md [hunk 1]
```diff
@@ -7,6 +7,21 @@ CurrentModule = MathOptInterface
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.13.0 (November 29, 2023)
+
+## Added
+
+ - Added [`ConstraintPrimalStart`](@ref) for [`Bridges.Constraint.GeoMeanBridge`](@ref)
+   (#2348)
+ - Added `verbose` keyword argument to [`Test.runtests`](@ref) (#2347)
+ - Added [`Bridge.FirstBridge`](@ref) attribute (#2211)
```
Comment (odow) [inline docs/src/changelog.md:17]:
```suggestion
 - Added [`Bridges.FirstBridge`](@ref) attribute (#2211)
```

**Comment by odow** [comment]
cc @pulsipher InfiniteOpt failing tests because of https://github.com/infiniteopt/InfiniteOpt.jl/pull/333.

**Comment by pulsipher** [comment]
Thanks for the quick PR fix!

**Comment by odow** [comment]
InfiniteOpt now passing, so this is good to go.

**Review by blegat — APPROVED** [review]

File: docs/src/changelog.md [hunk 2]
```diff
@@ -7,6 +7,21 @@ CurrentModule = MathOptInterface
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.13.0 (November 29, 2023)
```
Comment (odow) [inline docs/src/changelog.md:10]:
```suggestion
## v1.23.0 (November 29, 2023)
```

**Pull request merged**